scrape_data/scrape_data.py

```diff
@@ -44,21 +44,22 @@ def scrape(routes_df, url):
 
 def lambda_handler(event, context):
     API_KEY = os.environ.get("CHN_GHOST_BUS_CTA_BUS_TRACKER_API_KEY")
-    BUCKET_NAME = BUCKET_PUBLIC
     s3 = boto3.client("s3")
     api_url = (
         f"http://www.ctabustracker.com/bustime/api"
         f"/v2/getvehicles?key={API_KEY}"
     )
     routes_df = pd.read_csv(
```

Comment:
Not sure if this would improve things, but would it make sense to use `routes_df = pd.read_csv(s3.get_object(Bucket=BUCKET_PRIVATE, Key="current_routes.txt")["Body"]), chunksize=math.ceil(len(bus_routes) / 10))` here and then pass the chunks to `scrape`? Something like
```python
response_json = json.loads("{}")
for route in routes_df:
   response_json[f'chunk_{route}'] = scrape(route, api_url)
data = json.dumps(response_json)
```
where `scrape` is modified to return the request response.

Reply:
Not sure that this would be super useful -- this is not a big dataframe, I think loading the whole thing and then just iterating in the function is ok?

Reply:
Okay that sounds good then.